tests/jose/jws/general.spec.ts

```diff
@@ -195,16 +190,16 @@ describe('General JWS Sign/Verify', () => {
       resolve: sinon.stub().withArgs('did:jank:alice').resolves(mockResolutionResult)
     });
 
-    const verifier = new GeneralJwsVerifier(jws);
-
     const verifySignatureSpy = sinon.spy(Jws, 'verifySignature');
-    const cacheSetSpy = sinon.spy(verifier.cache, 'set');
+    const cacheSetSpy = sinon.spy((GeneralJwsVerifier as any).singleton.cache, 'set');
 
-    await verifier.verify(resolverStub);
-    await verifier.verify(resolverStub);
+    // intentionally calling verifySignature twice on the same JWS
+    await GeneralJwsVerifier.verifySignatures(jws, resolverStub);
+    await GeneralJwsVerifier.verifySignatures(jws, resolverStub);
+    await GeneralJwsVerifier.verifySignatures(jws, resolverStub);
```

Comment:
```suggestion
```

sanity: comment says twice, was there a reason to call it 3 times or just a mistake? It shouldn't affect the test outcome.